cristin-import/src/main/resources/possiblyEmptyFields.txt

```diff
@@ -30,6 +30,7 @@ entityDescription.reference.publicationContext.url
 entityDescription.reference.publicationContext.linkedContext
 entityDescription.reference.publicationContext.additionalIdentifiers
 entityDescription.reference.publicationContext.course
+entityDescription.reference.publicationContext.partOf
```

Comment:
Some test are failing since i added getter for partOf variable in MediaContribution class. I assume that this field can be empty.